selfdrive/assets/assets.qrc

```diff
@@ -13,5 +13,12 @@
   <file>offroad/icon_wifi_strength_medium.svg</file>
   <file>offroad/icon_wifi_strength_high.svg</file>
   <file>offroad/icon_wifi_strength_full.svg</file>
+  <file alias="languages.json">../ui/translations/languages.json</file>
+  <file alias="main_en">../ui/translations/main_en.qm</file>
+  <file alias="main_pt-BR">../ui/translations/main_pt-BR.qm</file>
+  <file alias="main_zh-CHS">../ui/translations/main_zh-CHS.qm</file>
+  <file alias="main_zh-CHT">../ui/translations/main_zh-CHT.qm</file>
+  <file alias="main_ko">../ui/translations/main_ko.qm</file>
+  <file alias="main_ja">../ui/translations/main_ja.qm</file>
```

Comment:
No way of doing this easily and programmatically? Right now we have everything nicely in one file for languages to show. Any advantages to adding them?

Reply:
Just want to embed the resource into the setup, so that the setup can be released  standalone without the language files.
I'm not sure how to improve it, feel free to modify it in a better way.